src/common/actions/repositories.js

```diff
@@ -5,8 +5,7 @@ import { conf } from '../helpers/config';
 
 const BASE_URL = conf.get('BASE_URL');
 
-export const FETCH_REPOSITORIES = 'FETCH_REPOSITORIES';
-export const FETCH_REPOSITORIES_SUCCESS = 'FETCH_REPOSITORIES_SUCCESS';
```

Comment:
Success is the same as set. I like the (fsa)[https://github.com/acdlite/flux-standard-action#example] pattern for actions like these:

```
type: FETCH_ALL_REPOSITORIES,
type: FETCH_ALL_REPOSITORIES, status: 'error', payload: error
type: FETCH_ALL_REPOSITORIES, status: 'success', payload: [ repos ] 
```

And I suggest use ALL here to distinguish actions which get only SNAP repos.